src/witness/myARG.ml

```diff
@@ -259,16 +284,8 @@ struct
     | None -> Arg.next n
 end
 
-module type MoveNode =
-sig
-  include Node
-
-  val move: t -> MyCFG.node -> t
-  val is_live: t -> bool
-end
-
-module Intra (Node: MoveNode) (ArgIntra: SIntraOpt) (Arg: S with module Node = Node):
-  S with module Node = Node =
+module Intra (ArgIntra: SIntraOpt) (Arg: S):
```

Comment:
Is this not `Inter`?

Reply:
Probably just because it connects the intraprocedural transformations with the ARG interface. But the latter is interproedural so it's confusing indeed.

If you have a better name suggestion, which makes the intra-inter connection clearer, it might be better indeed. Or maybe something like `ApplyIntra` or `MapIntra` as in applying/mapping the ARG with intraprocedural transformation.